### PR TITLE
Added new section

### PR DESCRIPTION
**Pull request by ProFire**
Added new section: Invalidating Individual Form Fields from Controller .
It will guide newbies who have a situation where they have to manually invalidate form fields without the use of validator class due to circumstances. The section is inline with the flow of the existing tutorial.

### CODE REVIEW

File: en/core-libraries/form.rst [hunk 1]
```diff
@@ -111,6 +111,32 @@ Once a form has been validated you can retreive the errors from it::
     ]
     */
 
+Invalidating Individual Form Fields from Controller
+==============================================
```
Comment (markstory) [inline en/core-libraries/form.rst]: This line should be as long as the title.

File: en/core-libraries/form.rst [hunk 2]
```diff
@@ -111,6 +111,32 @@ Once a form has been validated you can retreive the errors from it::
...
+It is possible to invalidate individual fields from the controller without the use of the Validator class. 
+The most common use case for this is when the validation is done on a remote server. 
+In such case, you must manually invalidate the fields accordingly to the feedback from the remote server.
+
+    // in src/Form/ContactForm.php
+    
+	public function setErrors($errors)
+	{
+		$this->_errors = $errors;
+	}
```
Comment (markstory) [inline en/core-libraries/form.rst]: Code blocks need to be indented.

File: en/core-libraries/form.rst [hunk 3]
```diff
@@ -111,6 +111,32 @@ Once a form has been validated you can retreive the errors from it::
     ]
     */
 
+Invalidating Individual Form Fields from Controller
+==============================================
+
+It is possible to invalidate individual fields from the controller without the use of the Validator class. 
+The most common use case for this is when the validation is done on a remote server. 
+In such case, you must manually invalidate the fields accordingly to the feedback from the remote server.
```
Comment (markstory) [inline en/core-libraries/form.rst]: Paragraphs that precede code blocks need to end in `::`.

File: en/core-libraries/form.rst [hunk 4]
```diff
@@ -111,6 +111,32 @@ Once a form has been validated you can retreive the errors from it::
...
+    
+	public function setErrors($errors)
+	{
+		$this->_errors = $errors;
+	}
+
+According to how the validator class would have returned the errors,
+$errors must be in this format: ["fieldName" => ["validatorName" => "The error message to display"]]
+
+Now you will be able to invalidate form fields by setting the fieldName, then set the error messages.
```
Comment (markstory) [inline en/core-libraries/form.rst]: Paragraphs that precede code blocks need to end in `::`.

**Comment by ProFire** [comment]
How about now?

**Comment by markstory** [comment]
Looking better, but you still need to indent the code blocks :smile:

**Comment by ProFire** [comment]
I'm confident I added a tab to the code blocks now. Is it ok?

**Comment by markstory** [comment]
Thanks @ProFire

**Pull request merged**